.github/FAQ.md

```diff
@@ -0,0 +1,99 @@
+## Fequently Asked Questions
+
+### Q1) What is a Test Annotation File?
+
+This is generally a file uploaded by a challenge host and is associated with a challenge phase. This file is used for ranking the submission made by a participant. An annotation file can be shared by more than one challenge phase. In the codebase, this is present as a file field attached to challenge phase model.
+
+### Q2) What to work on EvalAI?
+
+EvalAI’s issue tracker is good place to start. If you find something that interests you, comment on the thread and we’ll help get you started.
+Alternatively, if you come across a new bug on the site, please file a new issue and comment if you would like to be assigned. Existing issues are tagged with one or more labels, based on the part of the website it touches, its importance etc., which can help you select one.
+
+### Q3) What are the technologies that EvalAI uses?
+
+### Django
+Django is the heart of the application, which powers our backend. We use Django version 1.10.
+Django Rest Framework
+We use Django Rest Framework for writing and providing REST APIs. It's permission and serializers have helped write a maintainable codebase.
+
+### RabbitMQ
+We currently use RabbitMQ for queueing submission messages which are then later on processed by a Python worker.
+
+### PostgreSQL
+PostgresSQL is used as our primary datastore. All our tables currently reside in a single database named evalai.
+
+### Angular JS
+Angular JS is a well-known framework that powers our frontend.
```

Comment:
Can you please also add the version of Angular and also specify tasks runner that we are using.
For version, check package.json.